starknet_devnet/constants.py

```diff
@@ -13,7 +13,7 @@
 
 DEFAULT_ACCOUNTS = 10
 DEFAULT_INITIAL_BALANCE = 10**21
-DEFAULT_GAS_PRICE = 10**11
```

Comment:
I think this ought to be documented in the PR description usage changes. And run.md might has to be updated, too.

Reply:
added

Reply:
bbb8be7c6e73f8eb55148e5f596c093f420109ce